source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,571 @@
+About this Guide
+################
+
+********
+Audience
+********
+
+Leaders, Administrators, and/or Champions
+
+    **Note:** Some features described in this guide are only available to E10 and/or E20 (Enterprise Edition).
+
+*******************
+Learning Objectives
+*******************
+
+This guide provides foundational information necessary when developing a plan for a proof of concept and/or enterprise mobile deployment. 
+
+It will not dive into the technical implementation, but will reference any relevant documentation.
+
+By the end of this guide you should have an understanding of the high-level requirements for a successful Mattermost mobile deployment. 
+
+This includes…
+
+- Choosing how to obtain and distribute the Mattermost mobile applications
+- Determining the ideal mobile deployment model for your organization
+- Understanding the resource requirements when choosing to build your own Mattermost mobile apps
+
+*******
+Support
+*******
+
+If at any point you need additional help, we’re ready to assist, just reach out using the methods below…
+
+**Community**
+    Our entire team is active within our `public instance of Mattermost <https://community.mattermost.com/signup_user_complete>`_, plus you’ll have the support of one of the best open source communities around.
+**Documentation**
+    We link to a lot of `mobile-specific documentation <https://docs.mattermost.com/mobile/mobile-overview.html>`_ within this guide, but we encourage  you to check `all of our documentation <https://docs.mattermost.com/>`_.
+**Forums**
+    For more troubleshooting help, `open a new topic in our forums <https://forum.mattermost.org/c/trouble-shoot>`_ with steps to reproduce your issue. 
+**Enterprise Support**
+    If you’re an Enterprise Edition subscriber, you may open a support ticket in the `Enterprise Edition Support portal <https://mattermost.com/support/>`_.
+
+----
+
+################################
+Before You Begin Your Deployment
+################################
+
+A successful mobile deployment is an important part of your experience with Mattermost.
+
+For leaders, champions, and administrators there is a small window for success. As with any new project, a variety of risks can limit the success of adoption. This translates to wasted time, unnecessary cost, and unhappy teams. Any change, even beneficial, will have an impact on your users. 
+
+For this reason, our highest priority is making sure you have what you need to be successful. That starts with helping you identify the outcomes you’re looking to achieve.
+
+Another way of looking at this is to consider the goals for your mobile deployment.
+
+For example, do you want the simplest, fastest deployment possible?
+
+We have an option for that! Using the publicly available Mattermost mobile apps on the Google and Apple app stores can reduce a deployment to minutes.
+
+Better yet, this model can be rolled out to thousands of users in many different countries and languages?
```

Comment:
```suggestion
Better yet, this model can be rolled out to thousands of users in many different countries and languages.
```

Reply:
+1

Reply:
see above (already mentioned and will be solved)